packages/@aws-cdk/cdk/lib/stack.ts

```diff
@@ -42,7 +42,7 @@ export interface StackProps {
   readonly autoDeploy?: boolean;
 }
 
-const STACK_SYMBOL = Symbol('@aws-cdk/cdk.CfnReference');
+const STACK_SYMBOL = Symbol('@aws-cdk/cdk.Stack');
```

Comment:
IIUC we should use [`Symbol.for`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for). Otherwise, two versions of the library will have two different symbols, no?

Reply:
YES!!